docker-app/qfieldcloud/core/models.py

```diff
@@ -842,9 +842,13 @@ def clean(self) -> None:
         if self.organization.organization_owner == self.member:
             raise ValidationError(_("Cannot add the organization owner as a member."))
 
-        max_organization_members = self.organization.useraccount.current_subscription.plan.max_organization_members
+        plan_max_organization_members = self.organization.useraccount.current_subscription.plan.max_organization_members
+        max_organization_members = (
+            self.organization.useraccount.current_subscription.max_organization_members
+        )
+
         if (
-            max_organization_members > -1
+            plan_max_organization_members > -1
             and self.organization.members.count() >= max_organization_members
         ):
```

Comment:
Hm, I find the logic here a bit confusing. Basically we do the following:

check if the plan has `max_organization_members`
AND
check if the current organization has more than the subscription `max_organization_members`.

Would it make more sense to check for the `max_organization_members` in the same model in both conditions?